src/sqlparser/src/parser.rs

```diff
@@ -3133,6 +3133,28 @@ impl Parser<'_> {
         })
     }
 
+    /// BACKFILL_RATE_LIMIT = default | NUMBER
+    /// BACKFILL_RATE_LIMIT TO default | NUMBER
+    pub fn parse_alter_backfill_rate_limit(&mut self) -> PResult<Option<i32>> {
+        if !self.parse_keyword(Keyword::BACKFILL_RATE_LIMIT) {
```

Comment:
Oh, this explained the above question...Looks very ugly to me...But seems we still don't have a MV-level config map, then seems we don't have a better solution yet...

Reply:
Yeah I found it's kind of ugly too. I think maybe a refactor before this PR could be done. Let me pause this PR for a while. It's not urgent. We can cleanup the overwrite options first into options by MV/Sink/etc...

Reply:
I've taken a try and it seems kind of complex. Because we now have to add more options into the _optimizer_. Otherwise there's no way to construct the `stream_table_scan` with the `streaming_rate_limit` option.

Reply:
Sad, maybe we'd better leave it for future then. Thanks for trying tho👍